components/Slider/tests/snapshot/MDCSliderSnapshotTests.m

```diff
@@ -55,7 +55,7 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  // self.recordMode = YES;
+  //  self.recordMode = YES;
```

Comment:
Nit: revert this

Reply:
You're the second person who didn't like moving to the standard formatting for these comments.  Could you help me understand why you want this particular formatting (one space)?